src/main/asciidoc/reference-guide/tutorials/tutorials.adoc

```diff
@@ -17,9 +17,6 @@
 // under the License.
 //
 == Tutorials
+:hardbreaks:
```

Comment:
This is not needed.

Reply:
Which line do you mean?

Reply:
`:hardbreaks:`

Reply:
This was moved from a child-page of tutorial section up to tutorial main-page. Because tutorial child -pages use a lot of new lines to describe steps, I decided to move it up one level.